### ISSUE TITLE
install trouble SQL "Specified key was too long"

### ISSUE THREAD

**Issue by mquantin** [UIUCLibrary/teams#104]
Hello, 

I receive this error when installing. 

```
PDOException: SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes in /home/epotec/htdocs/omeka-s/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:53

Next Doctrine\DBAL\Driver\PDO\Exception: SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes in /home/epotec/htdocs/omeka-s/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDO/Exception.php:18

Next Doctrine\DBAL\Exception\DriverException: An exception occurred while executing '
CREATE TABLE team (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(240) NOT NULL, description LONGTEXT NOT NULL, UNIQUE INDEX UNIQ_C4E0A61F5E237E06 (name), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB;':

SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes in /home/epotec/htdocs/omeka-s/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:128
```
(full log attached [log.log](https://github.com/UIUCLibrary/teams/files/10489126/log.log) )

The table your plugin creates uses long characters? 
Tank you for your help.

**Comment by alexdryden** [comment]
Hi @mquantin,

Thanks for the report. Can you confirm which version of MySQL you are using? 

The default settings for MySQL on versions 5.6 and earlier enforced a 767 byte length for indexes on tables using InnoDB, so I suspect that is the root cause. Version 5.6 was EOL in Feb 2021, but it was very popular release and was the default for several package managers. 

If you are using MySQL version 5.6, the easiest fix would be to upgrade to 5.7 or later. If you cannot upgrade, I should be able to make a patch to support 5.6 later this week.

**Comment by mquantin** [comment]
Hi Alex, thanks for you answer. 
I did the update to mariaDB 10.5 (mysql 8)
I can now install the plugin. Great. 

but I cannot use it. I open a new issue for that: #108
